appknox/client.py

```diff
@@ -460,12 +472,16 @@ def rescan(self, file_id: int):
         Start a rescan for a file id
```

Comment:
`def rescan(self, file_id: int):` can you also add the return type annotation? 
(Its missing in upload_file & the new util method also)

Reply:
The return type will be simple `int` because we are not using `mapper_drf_api` in `upload_file` and `rescan` methods, right?